cli/tsc/dts/lib.deno_webgpu.d.ts

```diff
@@ -1397,7 +1661,57 @@ interface GPUCanvasConfiguration {
   colorSpace?: "srgb" | "display-p3";
   alphaMode?: GPUCanvasAlphaMode;
 }
-/** @category GPU */
+
+/**
+ * Provides a way to render WebGPU content to a canvas element.
+ *
+ * @example
+ * ```ts
+ * // Get a canvas element (either from DOM or creating a OffscreenCanvas)
+ * // Using OffscreenCanvas in Deno:
+ * const canvas = new OffscreenCanvas(800, 600);
```

Comment:
```
 deno
Deno 2.2.4
exit using ctrl+d, ctrl+c, or close()
REPL is running with all permissions allowed.
To specify permissions, run `deno repl` with allow flags.
> new OffscreenCanvas(800, 600)
Uncaught ReferenceError: OffscreenCanvas is not defined
    at <anonymous>:1:22
```

Reply:
Not sure how to add an example in for this one since canvas might not be available? Should the example create a texture directly?

Reply:
I think this exists to be used with `Deno.UnsafeWindowSurface`
cc @crowlKats

Reply:
removed